app/views/ReplacingLeadTrusteeView.scala.html

```diff
@@ -16,33 +16,37 @@
 
 @import controllers.routes._
 @import viewmodels.RadioOption
+@import views.html.components.{ErrorSummary, InputRadio, SubmitButton}
 
 @this(
-  main_template: MainTemplate,
-  formHelper: FormWithCSRF
+    main_template: MainTemplate,
+    formHelper: FormWithCSRF,
+    errorSummary: ErrorSummary,
+    inputRadio: InputRadio,
+    submitButton: SubmitButton
 )
 
 @(form: Form[_], leadTrustee: String, trustees: List[RadioOption])(implicit request: Request[_], messages: Messages)
 
 @main_template(
-  title = s"${errorPrefix(form)}${messages("replacingLeadTrustee.title")}"
+    title = s"${errorPrefix(form)}${messages("replacingLeadTrustee.title")}",
+    showBackLink = true
 ) {
 
     @formHelper(action = ReplacingLeadTrusteeController.onSubmit(), 'autoComplete -> "off") {
 
-        @components.back_link()
+        @errorSummary(form.errors)
```

Comment:
add options to the error summary so the error summary link works

Reply:
Done, checked the others and that's the only one I missed